### DISCUSSION TITLE
Fourth start menu option

### DISCUSSION THREAD

**Discussion by Nascarfast48** [Open-Shell/Open-Shell-Menu#1521] [Ideas]
In several Windows 10 betas (and I think in really, really, early versions of Classicshell that were available for Windows 8) there was an alternate start menu, which essentially combined elements of the windows vista-7 start menus and the windows 10 one. I've always liked this start menu layout, and would love to see it in Openshell.
Below is a picture I found on google with the start menu.
![image](https://user-images.githubusercontent.com/126414826/230258015-550b56e9-ddb8-4348-98f7-da528d33a417.png)

**Comment by JRDK92** [comment]
Looks like a great idea. Wonder how difficult it would be to create?
